src/dbnode/network/server/tchannelthrift/node/service.go

```diff
@@ -123,6 +123,24 @@ type serviceMetrics struct {
 	writeTaggedBatchRawRPCs tally.Counter
 	writeTaggedBatchRaw     instrument.BatchMethodMetrics
 	overloadRejected        tally.Counter
+
+	queryTimingFetchTagged  queryMetrics
+	queryTimingAggregate    queryMetrics
+	queryTimingAggregateRaw queryMetrics
+
+	queryTimingDataRead queryMetrics
+}
+
+func newQueryMetrics(name string, scope tally.Scope) queryMetrics {
+	return queryMetrics{
+		byRange: limits.NewQueryRangeMetrics(name, scope),
+		byDocs:  limits.NewCardinalityMetrics(name, scope),
+	}
+}
+
+type queryMetrics struct {
```

Comment:
can we move this into the limits package? Seems like we typically want the combo of by range / by cardinality together

Reply:
it doesn't have anything to do with limits though.

Reply:
actually i think it's strange they are in the limits package. moved them to the `node` package for the service.